content/chainguard/chainguard-images/how-to-use/images-directory/index.md

```diff
@@ -22,140 +22,178 @@ There are hundreds of Chainguard Containers available for use. To help users exp
+* **CVE ID**: the official identification number of each vulnerability present in the table
 * **Severity**: the severity of each given vulnerability. This can either read **Critical**, **High**, **Medium**, **Low**, or **Unknown**
-* **Package**: showing the package where the vulnerability was found
+* **Package**: the package where the vulnerability was found
 * **Version**: the version of the package containing the vulnerability
 * **Last detected**: the date and time when the vulnerability last appeared in a scan of the container image
 
-To the left of each row in the table is down-pointing chevron (**˅**). If you click on this symbol, more information about the given vulnerability will appear below the vulnerability's row.
+To the left of each row in the table is down-pointing chevron (**˅**). Clicking on this expands the row to show more information about the given vulnerability.
+
+<center><img src="imgs-dir-I.png" alt="Screenshot showing the expanded view of the 'CVE-2025-30258' vulnerability. This expanded view includes a brief description of the CVE as well as some reference links." style="width:1100px;"></center>
+<br />
 
-![Screenshot showing the expanded view of the "CVE-2024-41946" vulnerability. This expanded view includes a brief description of the CVE as well as some reference links.](imgs-dir-H.png)
+Specifically, this highlights the **Package** name and **Version** number of the package associated with the vulnerability. It also shows the **Fixed version** of the package, a brief **Description** of the vulnerability, and one or more **References** you can review to learn more about the vulnerability.
 
-Specifically, this will highlight the **Package** name and **Version** number of the package associated with the Vulnerability. It also shows the **Fixed version** of the package, a brief **Description** of the vulnerability, and one or more **References** you can review to learn more about the vulnerability.
+Please be aware that, as with SBOM data, Chainguard began generating vulnerability information for its images on November 15, 2023. For this reason, any versions of a given image that were released before that date won't have any vulnerability data to show. 
 
-Please be aware that, as with SBOM data, Chainguard began generating vulnerability information for its images on November 15, 2023. For this reason, any versions of a given image that were released before that date will not have any vulnerability data to show. 
+### Advisories
 
-### Advisories tab
+When you scan a newly-built Chainguard Container with a vulnerability scanner, the scanner typically won't report any CVEs. However, as software packages age, more vulnerabilities are reported and CVEs will begin to accumulate in images. When this happens, Chainguard releases security advisories to communicate these vulnerabilities to downstream container image users. You can find all the advisories issued for a given image in its Advisories tab. 
```

Comment:
I'm not sure this is quite right. It's probably overcomplicating it a little as well.

Maybe something like:

The advisories tab lists any CVEs that have affected the image in the past and what action Chainguard took to remediate the CVE.